src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/ProposedApisFeature.ts

```diff
@@ -7,26 +7,25 @@ import * as vscode from 'vscode';
 import * as vscodeapi from 'vscode';
 import { RazorDocumentManager } from './RazorDocumentManager';
 import { RazorDocumentSynchronizer } from './RazorDocumentSynchronizer';
-import { RazorLanguage } from './RazorLanguage';
 import { RazorLanguageServiceClient } from './RazorLanguageServiceClient';
 import { RazorLogger } from './RazorLogger';
-import { RazorDocumentSemanticTokensProvider } from './Semantic/RazorDocumentSemanticTokensProvider';
+
 export class ProposedApisFeature {
     constructor(
+        // @ts-ignore these aren't used for now
         private documentSynchronizer: RazorDocumentSynchronizer,
+        // @ts-ignore
         private documentManager: RazorDocumentManager,
+        // @ts-ignore
         private languageServiceClient: RazorLanguageServiceClient,
+        // @ts-ignore
         private logger: RazorLogger,
     ) {
     }
 
     public async register(vscodeType: typeof vscodeapi, localRegistrations: vscode.Disposable[]) {
         if (vscodeType.env.appName.endsWith('Insiders')) {
```

Comment:
I wanted to leave this logic in even though it does nothing so that there's less to re-work when we add the next proposed feature.